src/wasm.rs

```diff
@@ -799,6 +801,7 @@ where
     ///
     /// The `data` on `AppResponse` is data returned from `reply` call, not from execution of
     /// sub-message itself. In case if `reply` is not called, no `data` is set.
+    #[allow(deprecated)]
```

Comment:
Also consider moving disabling this check to parameters it refers to

Reply:
Moved this `#[allow(deprecated)]` as near the cause as possible.